src/test/java/com/google/sps/data/AvailabilityServletTest.java

```diff
@@ -67,6 +78,8 @@ public void validAvailabilityServletRequest() throws IOException {
   @Test
   public void invalidAvailabilityServletRequest() throws IOException {
```

Comment:
Add another test to check for what happens when someone is requestee for someone else (the 401 l mentioned above).

Reply:
Not applicable because the request does not include info about the user.